modules/OCPP201/OCPP201.cpp

```diff
@@ -765,6 +765,11 @@ void OCPP201::ready() {
     callbacks.configure_network_connection_profile_callback =
         [this](const ocpp::v201::NetworkConnectionProfile& network_connection_profile) { return true; };
 
+    callbacks.all_connectors_unavailable_callback = [this]() {
```

Comment:
Concerning naming: if the containing struct is already called `callbacks`, then I wouldn't add `_callback` at the end of all of its members.  But just by reading the name of this callback, `all_connectors_unavailable` I'm expecting that this callback is called, when all connectors are unavailable.   But then I'm not necessarily expecting that I should proceed with firmware installation